spec/integration/user_can_visit_business_index_page_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature "user can view business index" do
+  scenario "user visits home page and sees the business index link" do
+    visit root_url
+
+    expect(page).to have_link("Business Index")
```

Comment:
let's add a scope to this: within navbar or whatever the class we would expect is